typescript/cli/src/avs/check.ts

```diff
@@ -6,13 +6,14 @@ import {
   MerkleTreeHook__factory,
   ValidatorAnnounce__factory,
 } from '@hyperlane-xyz/core';
+import { ProtocolType } from '@hyperlane-xyz/provider-sdk';
 import {
   ChainMap,
   ChainName,
   MultiProvider,
   isValidValidatorStorageLocation,
 } from '@hyperlane-xyz/sdk';
-import { Address, ProtocolType, isObjEmpty } from '@hyperlane-xyz/utils';
+import { Address, isObjEmpty } from '@hyperlane-xyz/utils';
```

Comment:
i wonder if we should move also the "custom" types (like Address in this case) to the provider sdk and leave the utils package as a utility function only package

Reply:
On one hand, yes. But I can see how moving ProtocolType is making it impossible to see other changes on the same PR. So if we do more more types, it should be a dedicated change. Do you have any other from utils you wish to move?